docs/common/registration.md

```diff
@@ -113,6 +113,7 @@ To register a new resource, follow the instructions below:
    the [resource group template](https://github.com/opensciencegrid/topology/blob/master/template-resourcegroup.yaml)
    as a guide.
    You may leave any `ID` or `GroupID` fields blank.
+   When adding new entries, make sure that you do not change the formatting or indentation of the template.
```

Comment:
This makes it sound like they shouldn't modify the template file, which isn't what they're doing.